examples/blueprint-construct/index.ts

```diff
@@ -40,22 +40,24 @@ export default class BlueprintConstruct extends cdk.Construct {
         const prodBootstrapArgo = new ssp.addons.ArgoCDAddOn({
             // TODO: enabling this cause stack deletion failure, known issue:
             // https://github.com/aws-quickstart/ssp-amazon-eks/blob/main/docs/addons/argo-cd.md#known-issues
-            // bootstrapRepo: {
-            //      repoUrl: 'https://github.com/aws-samples/ssp-eks-workloads.git',
-            //      path: 'envs/dev',
-            //      targetRevision: "deployable"
-            // },
-            // adminPasswordSecretName: "argo-admin-secret"
+            bootstrapRepo: {
+                 repoUrl: 'https://github.com/aws-samples/ssp-eks-workloads.git',
```

Comment:
this will be removed since it will break e2e. Anytime we provision the app of apps, cluster then fails to destroy, because the CRD controller is dropped before the apps. I meant to look into it at some point (I believe we have an issue).